_posts/2015-02-03-webinar-on-ble.markdown

```diff
@@ -0,0 +1,39 @@
+---
+layout: post
+title:  "Webinar on BLE"
+date:   2015-02-03 12:30:14
+categories: rhc
+author: anuj
+---
+
+Last week, I conducted a hangout on air on behalf of Revealing Hour Creations. The title was "A look at Bluetooth Low Energy".  
+The motive was to introduce everyone to the technology and some of it's most sought out applications. It was more of a 101 for BLE than anything else.
+
+Here's the YouTube link to the Hangout on Air 
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/Jsci1RqkHHU?t=9m54s" frameborder="0" allowfullscreen></iframe>
+The topics that were covered in the webinar were  
+
+1. History of BLE  
+Where did the tech evolve, the people involved and the timeline
```

Comment:
I would prefer, there is content of what you spoke about here. Rather than just pointers (as in an agenda)